### DISCUSSION TITLE
essay/2023/09/29/give-back-more-than-take

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#132] [Comments]
# essay/2023/09/29/give-back-more-than-take

일론 머스크가 MZ세대, 취준생, 예비창업가들에게 주는 조언

https://jeho.page/essay/2023/09/29/give-back-more-than-take.html

**Comment by developedbyDK** [comment]
재호님 안녕하세요, 항상 글 잘 읽고 있습니다.
저도 재호님 말씀에 공감하며 1인개발자로서 작은 돈이라도 해외에서 수익창출을 하고자 작은 SaaS 크롬 익스텐션을 만들었습니다.그런데 결제 연동을 하려고하니 점점 글로벌 결제의 표준으로 자리잡고있는 Stripe의 경우 Live 계정 생성이 한국에서는 비활성화 되어있고, Paypal 결제는 가능한데 Paypal을 사용하자니 UI가 복잡하여 중간에 이탈이 염려되고 특히 페이팔은 한국 사용자 결제가 막혀있더군요. 제일 중요한 결제가 이래서야.. 국내 개발자들이 해외에 진출을 어떻게 하는지 염려가 되었습니다. (과기부는 무엇을 하는가!ㅠㅠㅠ) 혹시 재호님만의 해외 결제를 위한 노하우나 아이디어가 있는지 궁금합니다.

**Reply by BenjaminKim** [reply]
해외 결제 노하우는 저도 없습니다. 아직 해외 유저들에게 100원도 벌어보지 못해서요. ㅠㅠ
인앱결제를 사용하면 해외 결제가 전혀 문제 없긴합니다만, 크롬익스텐션은 좀 골치가 아프겠네요. Stripe가 가장 좋을 것 같긴한데, 말씀하신 문제는 저도 모르겠습니다. 해본 적이 없어서 ㅎ

**Reply by developedbyDK** [reply]
그렇군요. 답변 감사드립니다. 언젠가 스트라이프 결제도 지원됐으면 좋겠네요.

**Reply by BenjaminKim** [reply]
@developedbyDK [Polar.sh](http://polar.sh/) 라는 서비스가 [있나봐요](https://maily.so/josh/posts/g1o4k08grve). 한 번 살펴보셔도 좋을 것 같습니다. (저는 안 써봤지만요 ㅎ)

**Reply by developedbyDK** [reply]
> @developedbyDK [Polar.sh](http://polar.sh/) 라는 서비스가 [있나봐요](https://maily.so/josh/posts/g1o4k08grve). 한 번 살펴보셔도 좋을 것 같습니다. (저는 안 써봤지만요 ㅎ)

감사합니다. 시간나시면 제 서비스도 한 번 둘러봐주세요 ㅎㅎ 
url: https://chromewebstore.google.com/detail/glowtext-web-text-highlig/hdpffhckibfpohnmkgnebmoghofkigfb 
입니다.
